angular/src/lib/card/card.component.ts

```diff
@@ -8,9 +8,39 @@ import { Component, ElementRef, Input } from '@angular/core';
   styles: [],
   host: {
     'class': 'md-card',
+    '[class.md-card-clickable]': 'clickable',
+    '[attr.tabindex]': 'clickable ? "0" : null',
+    '(click)': 'clickable ? onClick() : null',
+    '(keydown)': 'clickable ? onKeydown($event) : null'
   }
 })
 
 export class CardComponent {
 
+  @Output() cardClicked: EventEmitter<Boolean> = new EventEmitter();
+
+  @Input() clickable: Boolean = false;
+
+  onClick(): void {
+    if (this.clickable) {
+      this.el.nativeElement.classList.add('active');
+      this.cardClicked.emit();
+    }
+  }
+
+  onKeydown(event: KeyboardEvent): void {
+    const key = event.key;
+    switch (key) {
+      case ' ':
```

Comment:
This only works if the logic is added that @daanishnasir added in:
```
    const key = event.key || event.which || event.keyCode;
```
Also might need to import '@angular/cdk/keycodes' for ENTER

Reply:
those are deprecated?  but I added them

Reply:
Agreed but .key isn't fully backwards compatible.